source/partials/release_notes/_release-20-9-0.md.erb

```diff
@@ -0,0 +1,68 @@
+<h4>Stage Overview on VSM Page</h4>
+
+Staring this release, the users can utilize the stage overview on VSM page. Earlier, it was available on the Dashboard and Pipeline Activity Page.
```

Comment:
users can utilize the stage overview on VSM page -> `users can view the stage overview on VSM page`